lib/screens/signup/signup.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:ecellapp/screens/signup/cubit/signup_cubit.dart';
+import 'package:ecellapp/screens/signup/widgets/email_field.dart';
+import 'package:ecellapp/screens/signup/widgets/first_name_field.dart';
+import 'package:ecellapp/screens/signup/widgets/last_name_field.dart';
+import 'package:ecellapp/screens/signup/widgets/password_field.dart';
+import 'package:ecellapp/screens/signup/widgets/mobile_number_field.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class SignupScreen extends StatelessWidget {
+  final _formKey = GlobalKey<FormState>();
+  final TextEditingController firstNameController = TextEditingController();
+  final TextEditingController lastNameController = TextEditingController();
+  final TextEditingController emailController = TextEditingController();
+  final TextEditingController passwordController = TextEditingController();
+  final TextEditingController mobileController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: BlocConsumer<SignupCubit, SignupState>(
+        listener: (context, state) {
+          if (state is SignupError) {
+            Scaffold.of(context).showSnackBar(
+              SnackBar(
+                content: Text(state.message),
```

Comment:
Try to lower the number of lines of code. It helps in easier code navigation and debugging.
```suggestion
                content: Text(state.message)
```
By removing this comma and formatting, the number of lines reduced by 2.